documentation/practices.md

```diff
@@ -41,18 +41,18 @@ If you use another CMS, you must update the configurations to reflect the public
 
 IBM Watson Content Hub is a cloud-based CMS. It also provides services with IBM Watson, such as cognitive tagging, to help transform assets into a searchable library of content.
 
-You can upload catalog images and site content images to WCH using the wchtools command-line utility. Uploaded assets are scanned using Watson's Visual Image recognition service, and tagged based on the content to create a searchable library. After the images are uploaded, the content delivery URL must be provided in the Reference Storefront's configuration for both `skuImagesUrl` and `siteImagesUrl`, with the appropriate sku/file name placeholders. In the following examples, a few fields are populated based on the tenant ID of the WCH account and the directory structure of the assets: 
+You can upload catalog images and site content images to WCH using the wchtools command-line utility. Uploaded assets are scanned using Watson's Visual Image recognition service, and tagged based on the content to create a searchable library. After the images are uploaded, the content delivery URL must be provided in the Reference Storefront's configuration for both `skuImagesUrl` and `siteImagesUrl`, with the appropriate sku/file name placeholders. In the following examples, a few fields are populated based on the tenant ID of the WCH account and the directory structure of the assets:
```

Comment:
Visual Image recognition service,
Why is V and I in caps? no comma required here.

After the images are uploaded, provide the content delivery URL in the Reference Storefront's configuration for both `skuImagesUrl` and `siteImagesUrl` with the appropriate sku/file name placeholders.

why do I feel, the following is correct :( : 
After the images are uploaded, provide the content delivery URL with the appropriate sku/file name placeholders in the Reference Storefront’s configuration for both `skuImagesUrl` and `siteImagesUrl` .